cl/opinion_page/templates/view_opinion_summaries.html

```diff
@@ -68,17 +68,20 @@ <h3>{{ summaries_count|intcomma }} judge-written summar{{ summaries_count|plural
               <ul class="group-summaries collapse" id="group-summaries-{{ representative.pk }}">
                 {% for summary in group.parentheticals.all %}
                   {% with describing_cluster=summary.describing_opinion.cluster %}
-                    <li class="v-offset-below-1">
-                      {{ summary.text|capfirst }}
-                      <br/>
-                      <span class="bullet-tail">{{ describing_cluster.date_filed }}</span>
-                      <span class="bullet-tail">
+                    {# Don't duplicate the representative cluster in the nested group #}
+                    {% if describing_cluster != representative_cluster %}
```

Comment:
Wait, actually on second look I think this could lead to a minor bug. Not because of the object comparison, but because multiple parentheticals can come from the same source cluster and this would filter all of them instead of just the representative. We want to check that `summary == representative` (or the corresponding primary keys), not that their clusters match

Reply:
Got it and fixed.